Education.feature

```diff
@@ -0,0 +1,16 @@
+﻿Feature: Education
+User wants to add education details into Profile Details
+Acceptance Criteria
+User is able to add the education details
+
+
+Scenario: 01 Test Add Education sucessfully
+	When User enter UniversityName as "Delhi University",CountryOfUniversity as "India",Title as "B.Tech", Degree as "Computer Science" and  YearOfGreaduation as "2012"  

```

Comment:
Missing a Given keyword. 

Every test case/specflow scenario needs to start with a login